solutions/instances/main.tf

```diff
@@ -166,3 +172,42 @@ module "scc_wp" {
   access_tags                   = var.scc_workload_protection_access_tags
   scc_wp_service_plan           = var.scc_workload_protection_service_plan
 }
+
+#######################################################################################################################
+# SCC Event Notifications Configuration
+#######################################################################################################################
+
+data "ibm_en_destinations" "en_destinations" {
+  count         = var.existing_en_crn != null ? 1 : 0
+  instance_guid = var.existing_en_crn
+}
+
+resource "ibm_en_topic" "en_topic" {
+  count         = var.existing_en_crn != null ? 1 : 0
+  instance_guid = var.existing_en_crn
+  name          = "SCC Topic"
+  description   = "Topic for SCC events routing"
+  sources {
+    id = local.scc_crn
+    rules {
+      enabled           = true
+      event_type_filter = "$.*"
+    }
+  }
+}
+
+resource "ibm_en_subscription_email" "email_subscription" {
+  count          = var.existing_en_crn != null ? 1 : 0
+  instance_guid  = var.existing_en_crn
+  name           = "Email for Security and Compliance Center Subscription"
+  description    = "Subscription for Security and Compliance Center Events"
+  destination_id = [for s in toset(data.ibm_en_destinations.en_destinations[count.index].destinations) : s.id if s.type == "smtp_ibm"].0
+  topic_id       = ibm_en_topic.en_topic.topic_id
+  attributes {
+    add_notification_payload = true
+    reply_to_mail            = "compliancealert@ibm.com"
```

Comment:
what is the email? Don't we need to ask consumer what value they want here?

Reply:
i'm going to make it an optional variables where consumers can change the email.